longest_palindromic_substring/longest_palindromic_substring.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+*  Implementation of dp problem: Longest Palindromic substring
+*  Time complexity: O(n^2)
```

Comment:
It would be better if the complexities are written, right above the function, which is implementing the algo.